configs/ci/roles/ros.json

```
@@ -0,0 +1,19 @@
+{
+    "roles": [
+        {
+          "name": "ROS Administrator",
```

For consistency on the platform, we'd ask that this be a lowercase `a` for `administrator`.
```suggestion
          "name": "ROS administrator",
```

---

sure